### DISCUSSION TITLE
Elasticity exercise - Halfplane under single load

### DISCUSSION THREAD

**Discussion by bricearnold** [gridap/Gridap.jl#679] [General]
Please i need your help i am new on Gridap: I want to computate the stress on a halfplane on single load F = 100N like on the picture. The Load should be on the middle of the halfplane. Please can you help me? I didn't found an example that correspond to my problem. Thank you

The cartesian mesh should be:
domain = (0,n,0,m)
partition = (4,4)
model = CartesianDiscreteModel(domain,partition)

The boundary condition should be u(y=0) = 0

And i wand to calculate the stress, so at the end i want the σ-tensor and the σ-value on a specific point when a force is loaded on the middle. 
Thank you![image](https://user-images.githubusercontent.com/81269766/138064374-4583804d-3ef1-4034-9f0d-aba7019014e5.jpeg)

**Comment by fverdugo** [comment]
Hi @bricearnold. The link seems to be broken.

**Reply by bricearnold** [reply]
I updated that, that was just a sketch of the task.
Thank for the mention!
